anta/cli/nrfu/commands.py

```diff
@@ -27,13 +28,19 @@
 @click.option("--tags", "-t", help="List of tags using comma as separator: tag1,tag2,tag3", type=str, required=False, callback=parse_tags)
 @click.option("--device", "-d", help="Show a summary for this device", type=str, required=False)
 @click.option("--test", "-t", help="Show a summary for this test", type=str, required=False)
-def table(ctx: click.Context, tags: Optional[List[str]], device: Optional[str], test: Optional[str]) -> None:
+@click.option(
+    "--group-by", default=None, type=click.Choice(["none", "host", "test"], case_sensitive=False), help="Group result by test or host. default none", required=False
+)
+def table(ctx: click.Context, tags: Optional[List[str]], device: Optional[str], test: Optional[str], group_by: str) -> None:
     """ANTA command to check network states with table result"""
+    console.print("\n")
```

Comment:
I was wondering where these newlines came from in my terminal :)
When using DEBUG mode, this line is irrelevant (at least from its original purpose).
Could you remove this and/or add it to the `print_settings()`/`print_table()` functions ?